src/components/atoms/result-display/result-display.component.tsx

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
-import './result-display.styles.scss';
+import Alert from '@mui/material/Alert';
```

Comment:
```suggestion
import { Alert } from '@mui/material';
```
nic takiego ważnego, często importuję w ten sposób z material'a, kiedy używam więcej niż jednego komponentu z material'a u siebie